src/sentry/middleware/ratelimit.py

```diff
@@ -33,5 +34,15 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         if rate_limit is None:
             return
 
-        if above_rate_limit_check(key, rate_limit)["is_limited"]:
+        rate_limit_check_dict = above_rate_limit_check(key, rate_limit)
+        if rate_limit_check_dict["is_limited"]:
             request.will_be_rate_limited = True
+            enforce_rate_limit = getattr(view_func.view_class, "enforce_rate_limit", False)
+            if enforce_rate_limit:
+                return HttpResponse(
+                    {
+                        "detail": f"You are attempting to use this endpoint too quickly. "
+                        f"Limit is {rate_limit_check_dict['limit']}/{rate_limit_check_dict['window']}s"
```

Comment:
This will we confusing when limit is 1 and window is 2:
```
Limit is 1/2s
```
maybe do
```py
f"Limit is {limit} requests per {window} seconds"
```